ampremover/ampremover.py

```
@@ -0,0 +1,192 @@
+        list
+            List of canonical URLs found
+        """
+        canonical_links = []
+        async with aiohttp.ClientSession() as session:
+            for url in urls:
+                api_url = f"https://www.amputatorbot.com/api/v1/convert?gac=true&md=3&q={url}"
+                try:
+                    async with session.get(api_url, timeout=10) as response:
+                        if response.status == 200:
+                            data = await response.json()
+                            if not 'error' in data:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)